TRIAGING.md

```diff
@@ -0,0 +1,80 @@
+## The Triaging Process at `conda`
+
+"Triaging" refers to the process of assessing the priority of incoming issues.
+
+In order to explain how various `conda` issues are evaluated, the following document will provide information about our triaging process in the form of an FAQ.
```

Comment:
Having this in a FAQ style is fine but I also think we should include a high-level diagram here that captures the gist. Something like:

```mermaid
flowchart LR
    issue_new(New Issues)

    subgraph board_triaging [Triaging Board]
    col_triaging{{Triaging}}
    col_support{{Support}}
    col_ready{{Ready}}
    col_closed{{Closed}}
    col_triaging<-->col_support-->col_closed
    col_triaging-->col_ready
    col_triaging-->col_closed
    end

    issue_new-->col_triaging
   col_ready-->Backlog
```